tests/test_main.py

```diff
@@ -15,6 +15,15 @@ def test_plain_email(self):
     def test_hash(self):
         self.assertEqual(gravify.Gravatar("bensoyka@icloud.com").hash, "7246821a7bf0b1b37794b39cb08ee052")
 
+    def test_hash_whitespace(self):
+        self.assertEqual(gravify.Gravatar("   \tbensoyka@icloud.com\n").hash, "7246821a7bf0b1b37794b39cb08ee052")
+
+    def test_hash_inner_spaces(self):
```

Comment:
You probably don’t need this one, as it isn’t a valid email.

Reply:
Just have spaces in the test, no tabs or newlines?

Reply:
Oh, I see you called out the inner spaces test. Sorry, I was confused with a different test. I'll remove it

Reply:
No not that, I’m saying you don’t need `test_hash_inner_spaces`, as that’s not a valid email.

Reply:
Sounds good.

Reply:
Test case removed